agent-tooling/src/main/java/io/opentelemetry/auto/tooling/context/FieldBackedProvider.java

```diff
@@ -403,10 +399,13 @@ public static void resetContextMatchers() {
           /*
            * For each context store defined in a current instrumentation we create an agent builder
            * that injects necessary fields.
+           *
+           * Does not use classLoaderMatcher() because those can be conflicting among
```

Comment:
If a future developer will read this code and will encounter this comment, will it be helpful or confusing? I mean, while reading the code, is it possible that developer will wonder about `classLoaderMatcher` at all? It looks to me that this comment answers the question that will not be asked, this increasing cognitive load, not decreasing it.

Reply:
makes sense, removed